test/TestHelpers.jl

```diff
@@ -216,6 +216,8 @@ end
     @inbounds deleteat!(parent(A), first_idx:last_idx)
 end
 
+Base.resize!(A::OffsetVector, nl::Integer) = resize!(A.parent, nl)
```

Comment:
> Why should resize! affect the end index but not the start index?

At least that's the definition of `resize!` given in the docstring: "If n is smaller than the current collection length, the first n elements will be retained." Why should the start index change?